addons/html_builder/static/src/core/utils.js

```diff
@@ -879,8 +876,8 @@ export function getAllActionsAndOperations(comp) {
             };
         }
     }
-    function getAllActions() {
-        const actions = getShorthandActions();
+    function getAllActions(onlyPreviewable = false) {
```

Comment:
Why do you need the onlyPreviewable ? Do you need to support a button with 1 previewable and 1 not previewable ?